data/releases/4.12.0.md

```diff
@@ -115,7 +115,7 @@ The user's manual for OCaml can be:
 - downloaded as a single
   [TAR](https://v2.ocaml.org/releases/4.12/ocaml-4.12-refman-html.tar.gz)
   or
-  [ZIP](https://v2.ocaml.org/releases/4.11/ocaml-4.12-refman-html.zip)
+  [ZIP](https://v2.ocaml.org/releases/4.11/ocaml-4.12-refman-html.zip - [404 Not Found])
```

Comment:
A better fix: 4.11/ocaml-4.12-refman-html.zip ⇒ 4.12/ocaml-4.12-refman-html.zip

Reply:
Thank you! I'll update that.